src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -520,6 +522,8 @@ def get_versioned_sdk(api_profile, resource_type, *attr_args, **kwargs):
     sub_mod_prefix = kwargs.get('mod', None)
     operation_group = kwargs.get('operation_group', None)
     sdk_path = get_versioned_sdk_path(api_profile, resource_type, operation_group)
+    if str(resource_type) == "ResourceType.MGMT_RESOURCE_TEMPLATESPECS":
+        sdk_path = 'azure.mgmt.resource.templatespecs.v2019_06_preview'
```

Comment:
> sdk_path = 'azure.mgmt.resource.templatespecs.v2019_06_preview'

This seems strange. Could we change the name of the package so that it can be obtained from the normal configuration logic?

Reply:
Yes, but this requires another SDK release and recording of tests, I have already requested the SDK release and started the process, but it is a long process so I will be releasing the change in a subsequent PR.

Reply:
The SDK was released (almost at the same time when you commented🙂): https://pypi.org/project/azure-mgmt-resource, so it might make sense to make the change. But I'm also fine if you do that in a separate PR, unless @zhoxing-ms thinks that's a blocker.